README.md

```diff
@@ -414,7 +414,7 @@ There is a WIP async Python client for Serial Keel.
 ### General structure
 
 ```python
-from serial_keel import connect
+from serialkeel import connect
 
```

Comment:
I read [here](https://peps.python.org/pep-0008/#package-and-module-names) that the use of underscores for module names is discouraged.

Reply:
Ok, good :slightly_smiling_face: